src/SharpBrick.PoweredUp/Enums/DeviceType.cs

```diff
@@ -15,7 +15,7 @@ public enum DeviceType : ushort
         ExternalTiltSensor = 0x0022, // TILT_SENSOR
         MotionSensor = 0x0023, // MOTION_SENSOR
         VisionSensor = 0x0025, // COLOR_DISTANCE_SENSOR
-        ExternalMotorWithTacho = 0x0026, // MEDIUM_LINEAR_MOTOR
+        MediumLinearMotor = 0x0026, // MEDIUM_LINEAR_MOTOR
```

Comment:
Naming choice is perfect. I copied these from poweredup-node ... but they have history.

Normally, this would apply for a breaking change. But honestly, the motor is not implemented so I cannot imagine why someone would use this enum value.

Therefore, fine with me!